src/common/android/packageNameResolver.ts

```diff
@@ -54,10 +54,10 @@ export class PackageNameResolver {
     }
 
     /**
-     * Gets the default package name, based on the applicaiton name.
+     * Gets the default package name, based on the application name.
      */
     private getDefaultPackageName(applicationName: string): string {
-        return PackageNameResolver.DefaultPackagePrefix + applicationName;
+        return PackageNameResolver.DefaultPackagePrefix + applicationName.toLowerCase();
```

Comment:
If you create an app with react-native init MyAppLication react-native will set the package name to:
com.myapplication

So you can go and modify files to make it case sensitive, but react-native seems to be suggesting for it to be lower case... So if we don't find the androidManifest.xml file, I'd personally go with the Lower Case version.